### PR TITLE
Rule-303: Amending rule 302

### PR DESCRIPTION
**Pull request by peteroestergaard**
Will update with details later: the idea is to implement Josh's informal suggestion when he proposed 302

### CODE REVIEW

**Comment by Snark1994** [comment]
You currently appear to be trying to merge into the rule-302 branch, whereas you probably want to be merging into the master branch (which now contains the changes from rule-302)

**Comment by cleborys** [comment]
That's of course my mistake (we branched from 302 to adopt Josh's changes). Is there a way to change the pull request retroactivel on the webpagey? I can't find one on mobile.
Otherwise, Peter,  you can close it and make a new edit :)

**Comment by Snark1994** [comment]
I have changed it to be merging into master - I think it will all work okay now.

**Comment by Snark1994** [comment]
I'm not sure if you're still planning on adding extra details later, but I would like it to specify e.g. that the proposer then has to publicly reveal the votes, and the votes must be revealed accurately.

**Comment by AnonymousVeNomicPlayer** [comment]
I agree with Snark1994 that there should be a few more details on the procedure of revealing the results added. Also, it might be good to specify the type of message more accurately. If I was about to send my favorite racing pigeon to you, that should maybe not be considered as a vote!?

**Comment by cleborys** [comment]
Please include an option to bet on Philipps racing pigeon.

**Comment by AnonymousVeNomicPlayer** [comment]
Cleborys, does your remark indicate that you do not trust in the reliability of my pigeons?

**Comment by malte-l** [comment]
I would prefer this rule to rule out using animals that did not consent for transmission of the message (this would probably include all carrier/racing pigeons, though I have full confidence in the reliability of AnonymousVeNomicPlayer's racing pigeons). [Also, I think carrier pigeons usually return to the place where they live or obtain food, so AnonymousVeNomicPlayer's pigeons can probably only be used to sen messages *to* them, and it seems unrealistic that AnonymousVeNomicPlayer can send their racing pigeon to someone else.]

**Comment by peteroestergaard** [comment]
Now I updated the rule as per the comments above (again, sorry for the slight delay). The current (and hopefully final) round of public voting by comment here is hereby open.

**Comment by AnonymousVeNomicPlayer** [comment]
VOTE: no

**Comment by AnonymousVeNomicPlayer** [comment]
Let me just say that I would in principle appreciate a rule change like this. The no vote is just for the following two reasons:
1. According to rule 218 (and 217) the final version of the proposal had to be ready by yesterday evening. I do not want to be overly pedantic here, but I am afraid that people could - if the proposal was accepted - claim later that we actually voted on the form in which it was yesterday evening. According to rule 122 the game state is independent of the git repository. So even when the new version of rule 303 is pushed to the git repository, someone could possibly claim that it really is the old version of this rule that applies. According to the old version, he would not have to reveal the votes, which could be a real problem if only a majority (and not unanimity) is required (since people then can't check whether their vote was counted correctly).
2. I am still not completely satisfied with the phrasing of the first two bullets in rule 303. They do not regulate precisely what is considered as a way to communicate a vote. In particular, I am afraid that people could try to be unreachable on the voting day, e.g. by changing their email address and claiming that they did not receive any votes, thereby only counting their own approving vote. I would be much happier if the rule would clearly state a way in which this communication should happen, e.g. "The proposer specifies an email address [early enough] to which the votes shall be send on the voting day".
Consequently I would be very happy to accept a rule change like the one proposed if the two points above are considered appropriately.

File: rules.md [hunk 1]
```diff
@@ -97,4 +97,7 @@ This rule takes precedence over every other rule determining the winner.
  * 219: Each player must have submitted their vote by the end of the first voting day following the first proposal day during a turn.
 
  * 303: The procedure for voting on rules is as follows and must be followed:
-    - send a private message to the proposer of the rule containing your vote.  The message must begin with the string "VOTE: " and everything after it will be considered part of the vote.
+    - each voter must send a private message to the proposer of the rule containing their vote.  The message must begin with the string "VOTE: " and everything after it will be considered part of the vote.
+    - each private message sent for this purpose must be sent via a means whereby the sender may hold the reasonable assumption that the proposer will receive the message by the end of the relevant voting day.
+    - the proposer of the rule holds the responsibility for collating the votes, and shortly after the end of the relevant voting day must publish the votes received accurately and truthfully, naming the voters alongside their vote, in a comment on the relevant branch. Only after this may the actions as described in rule 120 come into effect.
```
Comment (Snark1994) [inline rules.md]: I would rather "must accurately and truthfully publish the votes received, naming..."

**Comment by Snark1994** [comment]
VOTE: no

**Comment by Snark1994** [comment]
Similar feelings to Philipp; I agree that we could be argued to be currently voting on yesterday's version of the rule.

I also agree that the rule needs a little tightening. I think we could e.g. each have an officially designated email account for nomic purposes (not listed on the GitHub, of course, but which is distributed via other means and referred to in the rules) which is the only place to which a vote can be sent.

**Comment by peteroestergaard** [comment]
Thanks for the comments! Much better than just voting no without giving a reason :)

My feeling is that most of the concerns are covered by the existing rule that every player must have a vote for each proposal - I think this makes it the proposers responsibility to make sure they are contactable and to include all the votes they are sent.

I accept the criticism that the final version of my proposal was late - I thought I had gotten away with it since nobody had voted yet! I didn't think about how this might have some weird consequences in terms of the rule about the final version of the rule. Kicking myself for not being sufficiently organised - promise to do better next time ;)

**Comment by cleborys** [comment]
VOTE: yes
I like pigeons.

**Comment by malte-l** [comment]
VOTE: No

**Comment by malte-l** [comment]
I am also convinced by AnonymousVeNomicPlayer's arguments, I was also thinking of this problem with late changes before. I don't think that the rule with everyone having to vote results the proposer having to be reachable - in the previous rounds this was rather interpreted as as meaning that who did not vote on time votes yes or hasn't voted but is not counted for "unanimity" or something similar to this (given the difficulty of always having everyone vote on time that might also make sense...).
So I think requiring the proposer to specify an email address they have access to (and other players don't) to send votes to (and then actually checking it and revealing the votes truthfully and fully etc. etc.) before the voting day, and voting must happen by emails to this address is the way to go (and also gets rid of the potential for using carrier pidgeons).
I will think about this a bit more tomorrow and make a proposal starting from Peter's that addresses these issues.

**Comment by Smaug123** [comment]
Vote: no.

I like getting points.

**Pull request closed without merging**